### ISSUE TITLE
[GmailApi]Longer version of message.snippet

### ISSUE THREAD

**Issue by thirtyninetythree** [google/googleapis.dart#420]
A getter to return the text of an email like `message.snippet` instead of the base64 representation that it returns. Accessing it through the `payload.body.data` returns raw html which is hard to parse

**Comment by kevmoo** [comment]
We just generate these APIs. We have no direct knowledge of their behavior. The best we can do is to look at https://developers.google.com/gmail/api/
